lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -188,13 +189,29 @@ def sex(sex_name) -> str:
 
     return sex_map.get(sex_name, "other")
 
+
+def age(document: dict) -> str:
+    """
+    Given a *document*, retrieve age value and 
+    return as a string to fit the interval format.
+
+    If no value is given for age, then will just return None.
+    """
+    age = document.get("age")
+    if age is None:
+        return None
+    if age > 90:
+        return "90 years"
```

Comment:
The reason for this is that we want this kind of age masking to only happen outside of id3c. If we do it in the ETL process, then we've sort of hidden from ourselves that the unmasked ages exist in `receiving` because we won't see them in `warehouse`.